roles/pulp/tasks/main.yaml

```diff
@@ -16,15 +26,71 @@
     name: pulp-settings-py
     data: "{{ lookup('ansible.builtin.template', 'settings.py.j2') }}"
 
-- name: Deploy Pulp Container
+- name: Generate database symmetric key
+  ansible.builtin.command: "bash -c 'openssl rand -base64 32 | tr \"+/\" \"-_\" > /var/lib/pulp/database_fields.symmetric.key'"
+  args:
+    creates: /var/lib/pulp/database_fields.symmetric.key
+
+- name: Create database symmetric key secret
+  containers.podman.podman_secret:
+    state: present
+    name: pulp-symmetric-key
+    data: "{{ lookup('file', '/var/lib/pulp/database_fields.symmetric.key') }}"
```

Comment:
this looks up the file on the node Ansible running on (so "laptop"), but it's created inside the `quadlet` VM.

I changed this
```patch
+- name: Load database symmetric key
+  ansible.builtin.slurp:
+    src: /var/lib/pulp/database_fields.symmetric.key
+  register: pulp_key
+
 - name: Create database symmetric key secret
   containers.podman.podman_secret:
     state: present
     name: pulp-symmetric-key
-    data: "{{ lookup('file', '/var/lib/pulp/database_fields.symmetric.key') }}"
+    data: "{{ pulp_key['content'] | b64decode }}"
```

and that made it available properly